### PR TITLE
Add exec permissions for script

### PR DESCRIPTION
**Pull request by enriquebelarte**

### CODE REVIEW

File: training/intel-bootc/Containerfile [hunk 1]
```diff
@@ -85,7 +85,7 @@ COPY --from=libbuilder /opt/habanalabs/libfabric-1.20.0 /opt/habanalabs/libfabri
 COPY scripts/os_dependencies.sh /tmp/os_dependencies.sh
```
Comment (rhatdan) [inline training/intel-bootc/Containerfile]: Do the `--chmod` here

File: training/intel-bootc/Containerfile [hunk 2]
```diff
@@ -34,7 +34,7 @@ ARG REPO_TOKEN
 
 USER root
 COPY scripts/os_dependencies.sh /tmp/
```
Comment (rhatdan) [inline training/intel-bootc/Containerfile]: do `COPY --chmod=755 scripts/os_dependencies.sh /tmp`

Comment (enriquebelarte) [reply training/intel-bootc/Containerfile]: Wouldn't it be --chmod instead?

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**